packages/website/package.json

```diff
@@ -5,13 +5,16 @@
   "license": "MIT",
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
     "lint": "prettier --check . && next lint",
     "format": "prettier --write .",
-    "compileDocsForLLM": "node scripts/compileDocs.js"
+    "compileDocsForLLM": "node scripts/compileDocsForLLM.js",
+    "generateModulePages": "node scripts/generateModulePages.mjs",
```

Comment:
In `/hub`, I used `gen:*` naming convention for targets like these, it'd be good to use it here too.